source/slang/slang-check-conformance.cpp

```diff
@@ -145,15 +145,15 @@ namespace Slang
         }
     }
 
-    bool SemanticsVisitor::doesTypeConformToInterfaceImpl(
-        RefPtr<Type>            originalType,
-        RefPtr<Type>            type,
-        DeclRef<InterfaceDecl>  interfaceDeclRef,
+    bool SemanticsVisitor::_isDeclaredSubtype(
```

Comment:
> why does the name start with underscore?

As part of the coding conventions (not sure if this bit is documented currently), a method or function that starts with a `_` is conceptually an internal or implementation-detail function (independent of whether it is `protected`, `private`, etc.). This to help such functions stand out when reading the call sites (so you know whether you are using part of the official API of a type/subsystem), especially in cases where using something like `private` would make things a huge pain to work with (anything that starts to require `friend` declarations counts as a pain in my book...).

Reply:
> It is probably helpful in the long term if we can start to document what each function is doing and what does each parameter mean.

I agree that we should be better about documenting functions in the `.h` files. In this case since I was just renaming an existing function I didn't add a comment, but it probably would be a good idea to add one while I'm touching this code.